### DISCUSSION TITLE
cloud phase function

### DISCUSSION THREAD

**Discussion by oferyaron** [mitsuba-renderer/mitsuba3#505] [Q&A] [answered]
How do I introduce my phase function to a cloud ?
volume.vol, for example

**Comment by njroussel** [comment, marked as answer]
Hi @oferyaron 

I'm not quite sure I understand you're question. Are you trying to use a specific phase function in your scene description?
If so, you can take a look at the XML/Python examples [here](https://mitsuba.readthedocs.io/en/stable/src/generated/plugins_media.html#heterogeneous-medium-heterogeneous).

**Comment by oferyaron** [comment]
Thanks,
It looks like what I am looking for.

Ofer

בתאריך יום ה׳, 26 בינו׳ 2023, 12:55, מאת Nicolas Roussel ‏<
***@***.***>:

**Reply by oferyaron** [reply]
Thanks for help.
I am runing the Henyey-Greenstein option and it looks fine.
however I would also like to checj the Lookup table phase function option.
Is there an example for that option as well ?

**Reply by njroussel** [reply]
You can find a few here: https://github.com/mitsuba-renderer/mitsuba3/blob/master/src/phase/tests/test_tabphase.py
